### ISSUE TITLE
Define "component specifier"

### ISSUE THREAD

**Issue by vmx** [ipld/specs#335]
The [Schema Authoring Guide](https://github.com/ipld/specs/blob/34d7c29eb0049da18b1d5cc5a30c743d114e9da6/schemas/authoring-guide.md#schema-kinds) talks about "component specifier", but never defines what that is. I don't know what that is and [@mvdan neither](https://github.com/ipld/specs/blob/34d7c29eb0049da18b1d5cc5a30c743d114e9da6/design/history/exploration-reports/2020.09-learning-ipld.md#concepts-that-confused-me). Hence I open this issue.

**Comment by rvagg** [comment]
right, I think that's my awkward way of saying "field of a struct or type within a union". that whole section's language may need reviewing tbh.
